blogs/2020/07/27/containers-edu.md

```diff
@@ -0,0 +1,223 @@
+     * There is newly introduced [Docker Desktop support on Windows Home](https://docs.docker.com/docker-for-windows/install-windows-home/). It requires Windows 10, version 2004 and enabling the Windows Subsystem for Linux 2 (WSL 2) backend. Enable WSL 2 by following the [WSL 2 installation guide](https://docs.microsoft.com/windows/wsl/wsl2-install).
+     * For students who would prefer to not configure the WSL 2 backend, [Docker Desktop for Windows](https://docs.docker.com/docker-for-windows/install/#:~:text=System%20Requirements,Hyper%2DV%20on%20Windows%2010%3A&text=4GB%20system%20RAM) can alternatively be used on Windows 10 Pro, Enterprise, or Education (Build 16299 or later), and Hyper-V and Containers Windows features must be enabled.
+
+Let’s start off by launching VS Code, which we can do by typing `code .` in the command prompt (or just by selecting VS Code on your computer):
+
+![Launch VS Code from command prompt](2-code-ps.png)
+
+Once VS Code launches, ensure you’ve installed the [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension:
+
+![Remote containers extension](3-extension.png)
+
+When we install any of the Remote extensions, we’ll see the green Remote indicator on the bottom left of the Status bar:
+
+![Remote indicator in VS Code](4-remote-indicator.png)
+
+You can click on it to open the Command Palette and verify the Remote-Containers commands are listed:
+
+![List of remote commands in Command Palette](5-commands.png)
+
+### Accessing a container for your class
+
+Let’s walk through an example dev container to help students get a consistent coding environment. In our classroom, we could create a single GitHub repository to store exercises that share the same tech stack. For instance, all the Python assignments can use the same container and be stored in the same repo.
+
+We have an example [vscode-course-sample GitHub repo](https://github.com/microsoft/vscode-course-sample) with a Python dev container and two Python intro assignments. Let’s open it in VS Code.
+
+You can select the Remote indicator in the bottom left, or use the Command Palette, to bring up the Remote-Containers commands.
+
+Let’s call **Open Repository in Container...**
+
+![Open Repository in Container command](6-open-repo-command.png)
+
+We need to enter the URL to the GitHub repo where our container is stored, which in our case is https://github.com/microsoft/vscode-course-sample:
+
+![Paste GitHub URL in Command Palette](7-paste-repo-command.png)
+
+You can **Create a unique volume.** A volume is where files will be stored in our container:
+
+![Volume options in Command Palette](8-volume-command.png)
+
+Now that we’ve chosen our container repo, VS Code reloads to build the image and start the container:
+
+![VS Code reload to start container](9-vsc-reload.png)
```

Comment:
Suggestion: don't think we need both of these images. I'd stick to the close up image and get rid of this one.

Reply:
Thanks for the feedback- I was uncertain if having both images was a good idea or if just the close up was enough. I've removed the one indicated.